src/libopensc/internal.h

```diff
@@ -39,6 +39,7 @@ extern "C" {
 #include "libopensc/opensc.h"
 #include "libopensc/log.h"
```

Comment:
clang-format suggested the following change:


```
-#include "libopensc/opensc.h"
-#include "libopensc/log.h"
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/internal.h -->